chain/client/src/info.rs

```diff
@@ -71,6 +71,13 @@ impl InfoHelper {
         }
     }
 
+    pub fn chunk_processed(&mut self, shard: ShardId, gas_used: Gas) {
+        let teragas = 1_000_000_000_000f64;
```

Comment:
I suggest to use `chunk.gas_limit()` instead of hardcoding the constant

Reply:
To the best of my knowledge, the less we pre-compute stuff before putting it in prometheus the better, because it's the way the ecosystem is designed.

So IMO it would be better to have the metrics expressed in Tgas (to be vaguely human-readable without any loss of information), and then have a `chunk_gas_limit` metric (gauge) that we could then use to divide in grafana.

That said to avoid hardcoding one more constant, I just hoisted the previously-existing teragas constant to the top of the file, hopefully that looks ok? :)